pym/bob/input.py

```diff
@@ -428,9 +428,9 @@ def toStep(self, pathFormatter, upperPackage):
 class CoreStep(metaclass=ABCMeta):
     __slots__ = ( "package", "label", "tools", "digestEnv", "env", "args",
         "shared", "doesProvideTools", "providedEnv", "providedTools",
-        "providedDeps", "providedSandbox", "variantId", "sandbox", "sbxVarId" )
+        "providedDeps", "providedSandbox", "variantId", "sandbox", "sbxVarId", "packageRelocatable" )
```

Comment:
Maybe just name it "relocatable". We create millions of these objects and every byte counts. :wink:

Reply:
I think pythons compiler and interpreter is smart enough to optimize this.